### DISCUSSION TITLE
Octolamp build

### DISCUSSION THREAD

**Discussion by zacharycase** [martinwoodward/octolamp#5] [Show and tell]
Printed with white ASA and black ABS rather than PLA since that's what I had on hand
I'm also powering it with an external 5v power supply rather than through usb just to get the LEDs a bit brighter
![image](https://user-images.githubusercontent.com/80114515/214935299-c3a7fd9b-cdfb-4722-b649-19b8d67788ed.png)

**Comment by martinwoodward** [comment]
Looks wonderful in ASA and ABS.  Do I need to change the shape of the tail a bit to make it work better without supports you think?

**Reply by metcalfc** [reply]
I printed the cover flipped (i.e., octocat on the bed) on a lightly gluesticked smooth Prusa bed. Printed great.
